cob_mimic/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cob_mimic)
 
+add_compile_options(-std=c++11)
```

Comment:
for the same reason as in https://github.com/ipa320/cob_driver/pull/393#discussion_r290780814